storm-core/test/jvm/org/apache/storm/daemon/supervisor/BasicContainerTest.java

```diff
@@ -370,7 +370,7 @@ public void testLaunch() throws Exception {
         final String workerConf = ContainerTest.asAbsPath(log4jdir, "worker.xml");
```

Comment:
If you start the paths from `Files.createTempFile` instead of ``tmp` for `stormHome` and `stormLocal`, you should be able to get rid of the `prependFilePrefixIfOnWindows` method.